docs/features/tuples.work.md

```diff
@@ -0,0 +1,27 @@
+Tuples Work Items
```

Comment:
Right. Now that we have settled on target typing details we know most of the parts, so we can start specking it.
That can go in a separate change, I assume.